src/plib/views/scripts/index/domain-list.phtml

```diff
@@ -96,5 +96,45 @@
         }
 
         render();
+
+        function purchaseClick(event) {
+            if (this.classList.contains('disabled')) {
+                event.preventDefault();
+                return;
+            }
+
+            if (<?=$this->_isSymantecInstalled ? 'false' : 'true'?>) {
+                // Symantec is not installed
+                $$('div.sw-purchase a').each(function (a) {
+                    a.classList.add('disabled');
+                    a.update('<span class="ajax-loading">' + a.textContent + '</span>');
+                });
+                event.preventDefault();
+                var purchaseButton = this;
+                new Ajax.Request(window.location.pathname + 'index.php/index/install-symantec', {
```

Comment:
prepare URL in php code

Reply:
window.location.pathname could be equal to '/modules/security-advisor/index.php/index/domain-list/'